feat: start processing messages at latest offset (#67)
openedx/event-bus-kafka: CHANGELOG.rst

Unreleased
**********

[1.4.0] - 2022-10-21
********************

Changed
=======

> * Remove auto.offset.reset: earliest from consumer (new consumer groups will now consume only messages that are sent after the group was initialized)

Reviewer: Adjusted language to match the net result, rather than the fact that you technically removed a line.
```suggestion
* Changed "auto.offset.reset" from "earliest" to "latest" (the default value). New consumer groups will now consume only messages that are sent after the group was initialized.
```

> * Remove override of auto.offset.reset on consumer (which will default to "latest"). New consumer groups will consume only messages that are sent after the group was initialized.

Reviewer: I think this is heavy on the removal, which ultimately doesn't matter if we aren't even saying what was removed.  Why not something like:
```suggestion
* Updated auto.offset.reset on consumer to "latest" (the default value). New consumer groups will consume only messages that are sent after the group was initialized.
```
